### PR TITLE
chore: update documentation to reflect rebranding of fast-components-msft to fluentui

### PR DESCRIPTION
**Pull request by chrisdholt**
# Description
This PR updates documentation to reference the new package name being made here: https://github.com/microsoft/fluentui/pull/14345

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-foundation/src/anchor/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ sidebar_label: anchor
 custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-foundation/src/anchor/README.md
 ---
 
-`fast-anchor` is a web component implementation of an [HTML anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). The `fast-components` and `fast-components-msft` components support the same visual appearances as the button component (accent, lightweight, neutral, outline, stealth) as well as a hypertext appearance for use inline with text.
+`fast-anchor` is a web component implementation of an [HTML anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). The `fast-components anchor supports the same visual appearances as the button component (accent, lightweight, neutral, outline, stealth) as well as a hypertext appearance for use inline with text.
```
Comment (janechu) [inline packages/web-components/fast-foundation/src/anchor/README.md:8]:
```suggestion
`fast-anchor` is a web component implementation of an [HTML anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). The `fast-components` anchor supports the same visual appearances as the button component (accent, lightweight, neutral, outline, stealth) as well as a hypertext appearance for use inline with text.
```

Comment (EisenbergEffect) [inline packages/web-components/fast-foundation/src/anchor/README.md:8]: Looks like the ` is missing.

File: README.md [hunk 2]
```diff
@@ -64,22 +64,22 @@ This package does not export Web Components registered as [custom elements](http
...
 
 ### fast-components-msft
-**This package has been moved to [FluentUI](https://github.com/microsoft/fluentui/)**
+**This package has been moved to [FluentUI](https://github.com/microsoft/fluentui/) and has been renamed to `@fluentui/web-components`**
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Ffast-components-msft.svg)](https://badge.fury.io/js/%40microsoft%2Ffast-components-msft)
 
-`fast-components-msft` is a library of Web Components that *composes* `fast-foundation`. `fast-components-msft` uses the same custom element names as `fast-components`, but makes use of different stylesheets and may include components that specifically support Microsoft's Fluent design language.
+`@fluentui/web-components` is a library of Web Components that *composes* `@microsoft/fast-foundation`. `@fluentui/web-components` uses the same custom element names as `@microsoft/fast-components`, but makes use of different stylesheets and may include components that specifically support Microsoft's Fluent design language.
```
Comment (EisenbergEffect) [inline README.md:74]: Probably should reword this a bit so we don't say "uses the same custom element names". Maybe we can say "is compatible with". Need to change here and the other two places with similar content as well.

File: packages/web-components/fast-foundation/src/anchor/README.md (hunk 1 above)
Comment (chrisdholt) [reply packages/web-components/fast-foundation/src/anchor/README.md:8]: Gah - @JaneChu I did it again.

File: README.md (hunk 2 above)
Comment (chrisdholt) [reply README.md:74]: Yeah I change that elsewhere already, let me find it.

File: README.md [hunk 3]
```diff
@@ -64,22 +64,22 @@ This package does not export Web Components registered as [custom elements](http
...
+If you're looking to get started using our components right away, take a look at [the components quick start](http://fast.design/docs/components/getting-started). You'll also want to check out [our integrations](http://fast.design/docs/integrations/introduction) if you're looking to add the components into a Webpack build or incorporate them with another front-end framework. For those interested in implementing their own design system or customizing the styles of the components, after you [have a look at the components](http://fast.design/docs/components/getting-started), you'll want to read through [our styling docs](http://fast.design/docs/design/introduction). Finally, if your goal is to build your own components or apps with `@microsoft/fast-element`, you can learn all about that in our [guide to building web components with FASTElement](http://fast.design/docs/fast-element/getting-started).
```
Comment (awentzel) [inline README.md:82]: Should "components" usage be changed to Web Components for more specific designation?

Comment (chrisdholt) [reply README.md:82]: all we offer here is WC's, so I would say no.

**Pull request merged**